### DISCUSSION TITLE
the examples difference

### DISCUSSION THREAD

**Discussion by bouyeijiang** [hecrj/iced#1510] [Q&A]
The examples provided by the source code are inconsistent with the usage mode after release

**Comment by sengho66** [comment]
> The examples provided by the source code are inconsistent with the usage mode after release

I agree. It works perfectly well if you run any of the examples in the repo but not when you try to implement it on your own. I'm figuring out how to get them worked on my own program too.

**Comment by arjunair86** [comment]
I would recommend to clone the repo and use the path in dependency

**Reply by bouyeijiang** [reply]
Maybe that's the only way

**Comment by serzhiio** [comment]
Look at https://github.com/iced-rs/iced/discussions/1501

**Comment by lufte** [comment]
They should be consistent if you run them against the version of Iced for which they were written. If you downloaded Iced 0.4.2 with Cargo, then use the examples of the 0.4.2 tag of the repository.
